tests/unit/web/test_models.py

```diff
@@ -0,0 +1,57 @@
+from django.test import TestCase
+from django.core.files.uploadedfile import SimpleUploadedFile
+
+from web.models import Contact, Team
+
+
+class BaseTestCase(TestCase):
+
+    def setUp(self):
+        pass
+
+
+class ContactTestCase(BaseTestCase):
+
+    def setUp(self):
+        super(ContactTestCase, self).setUp()
+        self.contact = Contact.objects.create(
+            name='user',
+            email='user@domain.com',
+            message='test message',
+            )
+
+    def test__str__(self):
+        name = self.contact.name
+        email = self.contact.email
+        message = self.contact.message
+        final_string = '{0}: {1}: {2}'.format(name, email, message)
+        self.assertEqual(final_string, self.contact.__str__())
+
+
+class TeamTestCase(BaseTestCase):
+
+    def setUp(self):
+        super(TeamTestCase, self).setUp()
+        self.team = Team.objects.create(
+            name='user',
+            email='test@user.com',
+            description='An awesome description for Test User',
+            headshot=SimpleUploadedFile(
+                name='test_headshot.jpg',
+                content=open('frontend/src/images/deshraj.png', 'rb').read(),
+                content_type='image/png'
+            ),
+            github_url='www.github.com/testuser',
```

Comment:
Ha! I found this somewhere else, after I'm done here Ill put a PR for changing that too. :D